samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -9,4 +9,22 @@ tests:
       - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpunet
-    tags: ci_build
+    tags: ci_rs_build
```

Comment:
tag should be ci_build as original sample, you can add `ci_rs_build` as second tag for our use

Reply:
If you want to add second tag, please add it accordingly base which tag is used in RS cycle, and when we want to build it

Reply:
added new tags for integration and weekly